test/ex_machina/ecto_test.exs

```diff
@@ -10,13 +10,28 @@ defmodule ExMachina.EctoTest do
     end
   end
 
+  defmodule Article do
+    use Ecto.Schema
+    schema "articles" do
+      field :title, :string
+      belongs_to :author, User
```

Comment:
I think it would also be good to add another `belongs_to` that isn't defined in the factory to make sure there are no issues when the association is not set